core/src/main/kotlin/org/evomaster/core/output/service/RPCTestCaseWriter.kt

```diff
@@ -72,7 +72,7 @@ class RPCTestCaseWriter : WebTestCaseWriter() {
             if (rpcCallAction.response!=null && varName != null){
                 if (rpcCallResult.hasResponse())
                     handleAssertions(lines, varName, rpcCallResult)
-                else{
+                else if(!rpcCallResult.isExceptionThrown()){
```

Comment:
what shall happen if exection is thrown?

Reply:
if an exception is thrown, we will provide the info about what the exception is and where the exception happens as before and put the invocation under the try-catch. here I exclude the assertion for the response because with ind1 case study, the response is quite uncertain.